lib/uds_plus_test_kit/metadata.rb

```diff
@@ -0,0 +1,36 @@
+require_relative 'version'
+
+module UDSPlusTestKit
+  class Metadata < Inferno::TestKit
+    id :uds_plus
+    title 'UDS+ Test Kit'
+    description <<~DESCRIPTION
+      ## Documentation
+      - [Inferno documentation](https://inferno-framework.github.io/inferno-core/)
+      - [Ruby API documentation](https://inferno-framework.github.io/inferno-core/docs)
+      - [JSON API documentation](https://inferno-framework.github.io/inferno-core/api-docs)
+      - [UDS+ Implementation Guide](https://fhir.org/guides/hrsa/uds-plus/)
+      <!-- break -->
+
+      ## Connectathon Links
+      - [Connectathon Manager](http://conman.clinfhir.com/connectathon.html?event=con33)
+      - [Connectathon Manager Tutorial](https://www.youtube.com/watch?v=wBHHgZrSF-k)
+      - [FHIR 101 Video](https://vimeo.com/542197402/8fb80fea04)
+      - [UDS+ Connectathon Page](https://confluence.hl7.org/pages/viewpage.action?pageId=161056877)
+
+      ## Version
+      **Test Kit:** 1.1.0
+      **IG:** 1.1.2
+
+      ### Notes From the Developers
+      - There are known issues with validating certain codesets (not resource structure) when running this kit locally. If you are running locally and encounter an "Unknown Code" error for a given resource, try running that same resource on the hosted version of the kit and see if the codeset can be validated there.
+    DESCRIPTION
+    suite_ids [:uds_plus]
+    tags ['SMART-UDAP Harmonization']
```

Comment:
According to [here](https://inferno.healthit.gov/test-kits/smart-udap-harmonization/) the only tag should be `'UDAP'`

Reply:
Wait that was the wrong test kit - tags should be empty

<https://inferno.healthit.gov/test-kits/uds-plus/>